src/args.rs

```diff
@@ -225,6 +225,18 @@ pub struct CliArgs {
     )]
     pub web_upload_concurrency: usize,
 
+    /// chmod permissions of uploaded files
+    ///
+    /// This takes an octal number that is similar to the one the chmod use
+    #[cfg(unix)]
+    #[arg(
+        long = "chmod",
+        value_parser(parse_file_mode),
+        env = "MINISERVE_CHMOD",
+        default_value = "0600"
```

Comment:
So far, the default behavior was that we uploaded with default permissions of the underlying system. This changes it and that's a problem. If this option is not set, we shouldn't also apply permissions that differ from the system's default.

Reply:
> So far, the default behavior was that we uploaded with default permissions of the underlying system.

This isn't the case since 413a63a60307bdf60229670b0f858963604d62a3. The file is created with 0600 since then (and it is a problem for me, thus this pr).

I can change it default to what umask says.

Reply:
Oh crap. Well, looks like no one noticed. Might as well leave your change as it is, then.